stable/sonarqube/templates/ingress.yaml

```diff
@@ -1,7 +1,7 @@
 {{- if .Values.ingress.enabled -}}
 {{- $serviceName := include "sonarqube.fullname" . -}}
 {{- $servicePort := .Values.service.externalPort -}}
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
I think you mean: "networking.k8s.io/v1"

Reply:
also using https://helm.sh/docs/topics/chart_template_guide/builtin_objects/ may make it work in older and newer clusters.

Reply:
FYI, pending PR #19836 includes a fix that addresses @davidkarlsen's concern.